sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -104,6 +104,7 @@ export const AESTRoutesConst = {
   NOTICE_OF_ASSESSMENT_VIEW: Symbol(),
   REPORTS: Symbol(),
   APPLICATION_EXCEPTIONS_APPROVAL: Symbol(),
+  EXCEPTIONS: Symbol(),
```

Comment:
Can we follow the same one from the above one and call it `APPLICATION_EXCEPTIONS_PENDING` or similar?